fusedrug_examples/tests/test_bimodal_mca.py

```diff
@@ -50,6 +50,7 @@ def test_data_and_model(self) -> None:
         # forward pass
         lightning_module.forward(smiles, proteins)
 
+    @unittest.skip("Doesn't support Lightning >= 2.0.0")
```

Comment:
Is this the weird issue?
Can you open an issue and a trello card for alex?

Reply:
Yes it is.
Will do now 👍

Reply:
Done